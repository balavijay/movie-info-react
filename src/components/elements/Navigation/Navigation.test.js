import React from 'react';
import { shallow } from 'enzyme';


import Navigation from './Navigation';

describe ("Navigation Component " , () => {
   const  app = shallow(<Navigation />);
    test("Navigation render" , () => {
        expect(app).toMatchSnapshot();
    })

 
});