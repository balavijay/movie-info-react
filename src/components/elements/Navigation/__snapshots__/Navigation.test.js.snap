// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Component  Navigation render 1`] = `ShallowWrapper {}`;
